### ISSUE TITLE
[BUG] bpytop freezes after an automatic system time sync on a dual boot system

### ISSUE THREAD

**Issue by Kenivia** [aristocratos/bpytop#270]
**Describe the bug**

- bpytop freezes after an automatic system time sync 
- it responds to keypresses, but the graphs, clock, process list etc no longer updates(but still responds to sorting method changes, graph style changes etc)
- it resumes normally after a window resize

**To Reproduce**

This is done on a dual boot system with Windows 10 and Linux Mint
For context, without any [rather tedious modification](https://www.howtogeek.com/323390/how-to-fix-windows-and-linux-showing-different-times-when-dual-booting/), every time you boot into a different system(in this case, windows last time and Linux this time), the system time is reset to GMT. On windows, this requires a manual click of the time sync button, where Linux detects and fixes it automatically.

- set bpytop to start on boot with command `sh -c "gnome-terminal --maximize -- bpytop"`
- boot into Windows
- sync time in windows settings
- reboot into Linux
- wait for a few seconds for system to sync time
- bpytop freezes

Note that I was not able to reproduce this bug with manual timezone changes from date & time settings, as bpytop's clock did not respond to the manual changes which is also kind of a bug.
Also, I am in the timezone `GMT+13`, it might not work with timezones that are `GMT-x`

**Expected behavior**

bpytop doesn't freeze 

**Screenshots**

This screenshot is taken after the time sync has occurred and bpytop has frozen. As you can see the time on the top has stopped ticking for about 20 seconds.
Also, there is a weird line above the disks graphs and memory graphs, but that goes away after pressing Esc. I assume that is a separate bug. This seems to only occur at startup, as I can't reproduce it by running the command `sh -c "gnome-terminal --maximize -- bpytop"`.
![Screenshot from 2021-03-01 22-34-18](https://user-images.githubusercontent.com/47839308/109479475-49d62a00-7adf-11eb-9016-6e9a6001ceb4.png)

**Info (please complete the following information):**
 - bpytop version: 1.0.63 (but I've noticed this for a long time)
 - psutil version: 5.7.3
 - Linux distribution and version: Linux mint 20 Ulyana Cinnamon with kernel 5.8.18-050818-generic 
 - Terminal used: gnome terminal
 - Font used: Ubuntu Regular 10(I think)
 - Python version: 3.8.5 

Content of ~/.config/bpytop/error.log:
```
02/03/21 (11:10:35) | INFO: New instance of bpytop version 1.0.63 started with pid 2644
02/03/21 (11:10:35) | INFO: Loglevel set to DEBUG
02/03/21 (11:10:35) | DEBUG: Using psutil version 5.7.3
02/03/21 (11:10:35) | DEBUG: CMD: /usr/bin/bpytop
02/03/21 (11:10:35) | INFO: Config file version and bpytop version missmatch, will be recreated on exit!
02/03/21 (11:10:35) | DEBUG: Collect and draw completed in 0.314065 seconds
02/03/21 (11:10:36) | DEBUG: Init completed in 0.880018 seconds
```

**Comment by UmarJ** [comment]
I can reproduce this bug on my system by disabling network time synchronization and then changing the system clock.

```
$ timedatectl set-ntp 0
$ timedatectl set-time <any-time-in-the-past>
```
bpytop freezes even if you set your clock backward by a second. Interestingly, as soon as the system clock catches up with the bpytop clock, it unfreezes again. I am assuming this is because bpytop is waiting until a specific before it updates its boxes. There are no problems if you set your clock forward.

> Note that I was not able to reproduce this bug with manual timezone changes from date & time settings, as bpytop's clock did not respond to the manual changes which is also kind of a bug.

https://docs.python.org/3/library/time.html#time.tzset

**Comment by aristocratos** [comment]
@UmarJ 
Yeah, should probably be fixable by adding another check in the main loop. Just haven't had time to look at it yet.

The timezone bug I guess should be fixable by adding a condition to run `time.tzset()` once every hour or something like that...

**Comment by aristocratos** [comment]
@Kenivia 
Should be fixed in v1.0.66

@UmarJ 
Added timezone update once every hour in latest
